Standalone is no longer working in last build

`myrepo/package.json`
```js
   "dependencies": {
-    "cozy-konnector-libs": "^3.2.5",
+    "cozy-konnector-libs": "^3.7.2",
+    "cozy-jobs-cli": "^1.0.3",
+    "cozy-logger": "^1.0.3",
     "csv-parse": "^2.0.4",
     "moment-timezone": "^0.5.14"
   },
   "scripts": {
     "start": "node src/index.js",
-    "dev": "cozy-konnector-dev src/index.js",
-    "standalone": "cozy-konnector-standalone src/index.js",
-    "standalone:record": "REPLAY=record cozy-konnector-standalone src/index.js",
-    "standalone:replay": "REPLAY=replay cozy-konnector-standalone src/index.js",
+    "dev": "cozy-run-dev src/index.js",
+    "standalone": "cozy-run-standalone src/index.js",
+    "standalone:record": "REPLAY=record cozy-run-standalone src/index.js",
+    "standalone:replay": "REPLAY=replay cozy-run-standalone src/index.js",
```

Running `yarn standalone` lead to this error:

```sh
Error: Cannot find module 'cozy-konnector-libs/helpers/init-konnector-config'
[…]
    at Object.<anonymous> (<MYREPO>/node_modules/cozy-jobs-cli/src/standalone.js:17:16)
```

This is fixed by patchin `cozy-jobs-cli/src/standalone.js`

```js
17- const config = require('cozy-konnector-libs/helpers/init-konnector-config')()
17+ const config = require('./init-konnector-config')()                              
```

And  it gets stuck right after:

```js
Error: Cannot find module './init-konnector-config'
[…]
    at Object.find (<myrepo>/node_modules/cozy-konnector-libs/helpers/cozy-client-js-stub.js:57:24)
```

This line is responsible for this:

```js
        const config = require('./init-konnector-config')()
```

I don't know `lerna`. I don't know if there is a way to share this same file between the two packages.

--- a/packages/cozy-jobs-cli/src/standalone.js
+++ b/packages/cozy-jobs-cli/src/standalone.js
@@ -14,7 +14,7 @@ process.env.COZY_FIELDS = JSON.stringify({
   folder_to_save: '.'
 })
 
-const config = require('cozy-konnector-libs/helpers/init-konnector-config')()
+const config = require('./init-konnector-config')()
 process.env.COZY_URL = config.COZY_URL
 
 const filename = program.args[0] || process.env.npm_package_main || 'index.js'

--- a/packages/cozy-konnector-libs/helpers/cozy-client-js-stub.js
+++ b/packages/cozy-konnector-libs/helpers/cozy-client-js-stub.js
@@ -54,7 +54,8 @@ module.exports = {
       // exeption for "io.cozy.accounts" doctype where we return konnector-dev-config.json content
       let result = null
       if (doctype === 'io.cozy.accounts') {
-        const config = require('./init-konnector-config')()
+        const configPath = path.resolve('konnector-dev-config.json')
+        const config = require(configPath)
         result = {auth: config.fields}
       } else {
         return Promise.reject(new Error('find is not implemented yet in cozy-client-js stub'))

